test/fixtures/linter/autofix/bigquery/004_templating/after.sql

```diff
@@ -66,7 +66,7 @@ imputed_effect_sizes AS (
         , COALESCE(IF(IS_NAN(o.{{metric}}_{{action}}), 0, o.{{metric}}_{{action}}), 0) AS {{metric}}_{{action}}
         , COALESCE(IF(IS_NAN(n.{{metric}}_{{action}}), 0, n.{{metric}}_{{action}}), 0) AS new_{{metric}}_{{action}}
         , n.campaign_count_{{action}}
-    {% endfor %}
+        {% endfor %}
```

Comment:
This test uses L001, and the output changed -- now L001 leaves this line alone rather than messing up the indentation.

Reply:
Good improvement!